terraform/bastion/Jenkinsfile

```diff
@@ -9,6 +9,9 @@ pipeline {
     choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are creating the instance in")
     choice(name: "COMMAND", choices: ["apply", "destroy"], description: "Apply will create the instance")
     text(name: "PUBLIC_KEY", description: "Public key. This is not needed if you're not going to use ssh tunnelling")
+    booleanParam(name: 'CONNECT_TO_BACKEND_CHECKS_EFS', defaultValue: false, description: 'Allow the bastion to connect to the backend checks EFS volume')
```

Comment:
Does one of these have to be set to `true` for the job to work? What happens if more than one is set to `true`?

Reply:
They can all be set to false but you'll be left with a bastion that can't connect to anything. If you select more of them, you get access to whichever ones you've checked.

Reply:
Ah, that makes sense.